lib/Data/Mapper/LanguageMapper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\Data\Mapper;
+
+use eZ\Publish\API\Repository\Values\Content\Language;
+use eZ\Publish\API\Repository\Values\ValueObject;
+use EzSystems\RepositoryForms\Data\Language\LanguageCreateData;
+use EzSystems\RepositoryForms\Data\Language\LanguageUpdateData;
+
+class LanguageMapper implements FormDataMapperInterface
+{
+    /**
+     * Maps a ValueObject from eZ content repository to a data usable as underlying form data (e.g. create/update struct).
```

Comment:
I overall wouldn't repeat the interface's / parent's phpdoc unless it adds value.